corehq/apps/export/views/download.py

```diff
@@ -519,6 +521,33 @@ def parent_pages(self):
         }]
 
 
+@location_safe
+class DownloadNewDatasourceExportView(BaseProjectDataView):
+    urlname = "data_export_page"
+    page_title = gettext_noop("Export Datasource Data")
+    template_name = 'export/datasource_export_view.html'
+
+    def dispatch(self, *args, **kwargs):
+        if not EXPORT_DATA_SOURCE_DATA.enabled(self.domain):
```

Comment:
Nit: Instead of checking if the toggle is enabled here, you can use the `@toggles.EXPORT_DATA_SOURCE_DATA.required_decorator()` decorator on the class itself.